src/services/api/index.ts

```diff
@@ -122,4 +122,24 @@ export default class ApiService {
     };
     return doRequest(url, 'POST', body, { apikey });
   }
+
+  static async fetchParticipantsInGroup (
+    apiKey: string,
+    groupId: string,
+  ) {
+    const path = `/groups/participants/${groupId}`;
+    const baseUrl = config.get<string>('apiUrl');
+    const url = this.createUrl(baseUrl, path);
+    return doRequest(url, 'GET', undefined, { apikey: apiKey });
+  }
```

Comment:
You don't need to get the API key by parameter, you have it in the config object.

Reference: https://github.com/SuperViz/sdk/blob/main/src/services/config/types.ts#L8

Reply:
```suggestion
  static async fetchParticipantsByGroup (
    groupId: string,
  ) {
    const path = `/groups/participants/${groupId}`;
    const baseUrl = config.get<string>('apiUrl');
    const url = this.createUrl(baseUrl, path);
    return doRequest(url, 'GET', undefined, { apikey: config.get('apiKey') });
  }
```